components/Form/Select/Select.js

```diff
@@ -115,6 +115,7 @@ export default function Select({
       <div className={styles.selectFeedbackGrouping}>
         <ThemedReactSelect
           {...props}
+          closeMenuOnSelect={false}
```

Comment:
Looks like this is the right prop, but you'll wanna set it on the source code for ThemedReactSelect and only if `isMulti` is true.

Reply:
ah yep, I'll do that